ERCS/erc-7638.md

```diff
@@ -0,0 +1,190 @@
+---
+eip: 7638
+title: Optimized Calls Encoding
+description: Encode mutiple calls for smart contract wallet with less gas
+author: George (@JXRow), Zisu (@lazy1523)
+discussions-to: https://ethereum-magicians.org/t/erc-7638-optimized-calls-encoding/18966
+status: Draft
+type: Standards Track
+category: ERC
+created: 2024-02-26
+---
+
+## Abstract
+`Optimized Calls Encoding`(OCE) outlines a solution for Smart Contract Account(SCA) wallets to consolidate multiple calls into a single call, encoding multiple parameters into bytes, compressing on-chain data, and saving gas. It can be used to implement atomic operations as well as non-atomic operations.
```

Comment:
Only use backticks (\`) for code snippets please

```suggestion
Optimized Calls Encoding (OCE) outlines a solution for Smart Contract Account(SCA) wallets to consolidate multiple calls into a single call, encoding multiple parameters into bytes, compressing on-chain data, and saving gas. It can be used to implement atomic operations as well as non-atomic operations.
```